### DISCUSSION TITLE
Run code button is not working

### DISCUSSION THREAD

**Discussion by celestefaz** [community/community#130527] [GitHub Education]
### Hi everyone,

Hi, 

I am trying to work on an assignment for one of my classes, and I have reached out to my school's technical support, but they will not help me. Whenever I type the code in and click run code, it does not run. When I look at the test feedback, it tells me "no attempts made," as seen in the image. I am seriously at a loss. Thank you for any help in advance. 
![Screenshot 2024-06-30 172837](https://github.com/community/community/assets/172634295/ec771fb2-1f42-4836-b107-426210634b35)

**Comment by asdacosta** [comment]
Try these:

> Check for syntax errors. You can try running another code that you’re sure works. I’m not familiar with SQL so can’t help debug. 
> Clear cache and re-run the code.
> If you’re not confined to use specific coding environment, you can try another one.

**Comment by Rishi-Sudhakar** [comment]
The reason is you didn't run it.
You should select the test, using mouse, then click on run. 
SQL will only run the text you select.

**Reply by celestefaz** [reply]
I did run it. This is what I got after I ran it. It doesn't generate a grade for me.. it shows that I didn't do an attempt. 
<img width="959" alt="task 01" src="https://github.com/community/community/assets/172634295/a4d333e8-3daa-40fa-8b4b-c423a94ee8d6">

**Reply by Rishi-Sudhakar** [reply]
Interesting, 

1. Check if you need to do any commits/pushes so that the grade will be updated. (you can either use the version control at the left bar, or use terminal to do the same, 
```
git add .
```
```
git commit -m "some commit"
```
```
git push -u origin main
```
main/master branch. If you don't know which branch you're in you can just use,

```
git branch
```
(So what this "pushing" does is, it will push your code to the remote url in which you are working)

2. If it doesn't work, maybe try using terminal to run sql queries.
